stacks/azure/application/centraldashboard/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../common/centraldashboard/overlays/stacks
+
+patchesStrategicMerge:
+- links-configmap.yaml
```

Comment:
`links-configmap.yaml` should be a resource.  What you need is to patch deployment as I understand

Reply:
Thanks. I'll just copy the approach you took instead.